### DISCUSSION TITLE
I have no idea how to connect to services with EMQX in Azure AKS Kubernetes

### DISCUSSION THREAD

**Discussion by xtianus79** [emqx/emqx-operator#391] [Q&A]
From the looks of it and the countless days and documentation readings it seems as if the pods in my AKS subnet cannot be configured without breaking the broker. 

No plugins or modules work when setting the URL. 

I have a deployed service over 443 that can be clearly reached inside of the emqx pod(s) via curl but setting the conf such as redis or such as emqx_auth_http.conf will absolutely not work when changing the url to a service inside of the subnet or over the internet via 80 or 443. 

What am I missing. It doesn't make any sense to me. 

i've posted an issue here. 

https://github.com/emqx/emqx/issues/8977

But more of a broader question. Where is the documentation and what could I possibly be missing that this is not working? 

I would think that my services could exist via in the subnet and or via 443/80.

**Comment by xtianus79** [comment]
The plugin works. Thanks to this information I was able to get it working. However, the modules are still broken. 

https://github.com/emqx/emqx-operator/blob/main/config/samples/emqx/v1beta3/emqx-plugin.yaml

**Comment by Rory-Z** [comment]
Hi, @xtianus79 Looks like some problems with EMQX, I already ping EMQX developer to check that, thx

**Comment by Rory-Z** [comment]
Hi @id Looks https://github.com/emqx/emqx/issues/8977 is closed, anything update ?

**Comment by id** [comment]
@Rory-Z added to roadmap for v5.0.24. Unless something urgent takes priority, we'll look into it in the next 2-3 weeks.

**Reply by Rory-Z** [reply]
Cool, if have any update, please let me know.
